codex-ui/src/vue/switch/Switch.types.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * States of the switch
+ */
+export type SwitchStates = 'hover' | 'default' | 'disabled';
```

Comment:
What the difference between this states?

Reply:
this states are used to calculate css class for element